### ISSUE TITLE
Symfony 3.4 deprecation warning abut Auto-registration of command

### ISSUE THREAD

**Issue by TerjeBr** [shivas/versioning-bundle#29]
I get this deprecation warning in my log:

```
[2018-01-08 16:27:54] php.INFO: User Deprecated: Auto-registration of the command "Shivas\VersioningBundle\Command\VersionBumpCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Auto-registration of the command \"Shivas\\VersioningBundle\\Command\\VersionBumpCommand\" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead. at /var/www/brokerportal/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Bundle/Bundle.php:190)"} []
```

**Comment by shivas** [comment]
give a try to new 3.0.0 version of bundle

**Comment by shivas** [comment]
Fixed in release  2.0.1 thanks to @mdriessen
